### PR TITLE
ZK-4505: resize() doesn't reduce a groupbox height with title

### PR DESCRIPTION
**Pull request by b952426**
ZK-4505: resize() doesn't reduce a groupbox height with title

### CODE REVIEW

File: zktest/src/archive/test2/config.properties [hunk 1]
```diff
@@ -2875,6 +2875,7 @@ B90-ZK-4431.zul=A,E,Multislider
 ##zats##B95-ZK-4699.zul=A,E,Listbox,Listitem,focus
 ##manually##B95-ZK-4659.zul=A,E,Tree,ROD,loading
 ##zats##B95-ZK-4567.zul=A,E,Frozen,scroll
+##zats##B95-ZK-4505.zul=A,E, resize,vflex,min,grouopbox
```
Comment (scribetw) [inline zktest/src/archive/test2/config.properties:2878]: ,resize,

File: zktest/test/java/org/zkoss/zktest/zats/test2/B95_ZK_4505Test.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+/**
+ * @author katherinelin
+ */
+public class B95_ZK_4505Test extends WebDriverTestCase {
+	@Test
+	public void test() throws InterruptedException {
+		connect();
+		click(jq("@button").eq(0));
+		waitResponse();
+		assertEquals(jq(".z-button").outerHeight(), jq(".z-groupbox-content").height());
```
Comment (scribetw) [inline zktest/test/java/org/zkoss/zktest/zats/test2/B95_ZK_4505Test.java:26]: jq(".z-button:eq(0)")
jq(".z-groupbox-content:eq(0)")

**Review by scribetw — CHANGES_REQUESTED** [review]

**Review by plijyfes — APPROVED** [review]

**Review by DevChu — APPROVED** [review]

**Comment by DevChu** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Review by scribetw — APPROVED** [review]

**Pull request merged**